mlflow/deployments/cli.py

```diff
@@ -63,6 +65,20 @@ def _user_args_to_dict(user_list):
     "list of supported config options.",
 )
 
+parse_input = click.option(
```

Comment:
Could we copy the API & parameter descriptions of `mlflow models predict`? There the options look like:

```
  -i, --input-path TEXT    CSV containing pandas DataFrame to predict against.
  -o, --output-path TEXT   File to output results to as json file. If not
                           provided, output to stdout.
```

Reply:
Done. Copied the parameters and descriptions from `mlflow models`